lua/kickstart/plugins/debug.lua

```diff
@@ -0,0 +1,69 @@
+return {
+  {
+    enabled = true,
+    config = function()
+      -- Optional debug adapter setup
+      --
+      -- To enable setup, change `disable = true` in the packer section
```

Comment:
Guessing you'll want to update 'in the packer section' to something lazy.nvim specific?

Reply:
yeah, i will make it so the debug config works with lazy -- this was leftover from an old PR that I never submitted